crates/bevy_scene/src/scene.rs

```diff
@@ -47,7 +50,9 @@ impl Scene {
         Ok(Self { world: new_world })
     }
 
-    /// Write the entities and their corresponding components to the given world.
+    /// Writes all resources/entities and their corresponding components in the scene world to new
+    /// entities in the given world. On success returns an instance mapping entities in the scene world
```

Comment:
Style: the second sentence should have an empty line, and then be on its own line :)